Sharpmake.Platforms/Sharpmake.CommonPlatforms/Android/AndroidPlatform.Vcxproj.Template.cs

```diff
@@ -18,7 +18,11 @@ public static partial class Android
         public sealed partial class AndroidPlatform
         {
             private const string _projectStartPlatformConditional =
-    @"  <PropertyGroup Label=""Globals"" Condition=""'$(Platform)'=='ARM64' Or '$(Platform)'=='x64' Or '$(Platform)'=='ARM' Or '$(Platform)'=='x86'"">
+    @"  <PropertyGroup Label=""Globals"" Condition=""'$(Platform)'=='ARM64' Or '$(Platform)'=='ARM'"">
```

Comment:
Ideally this conditional string should be fabricated with the list of platforms present in the vcxproj, not be hardcoded!

Reply:
I made it dynamically add all the platforms that are referenced using android.